app/services/stacks_file_resolver.rb

```diff
@@ -4,6 +4,7 @@ def initialize(base_path:)
   end
 
   def pattern(id)
+    id = id.gsub('%2F', '/') if Settings.server.allow_encoded_slashes == 'NoDecode'
```

Comment:
Could you use `URI.decode(id)`?

Reply:
I thought about it; my understanding of `allow_encoded_slashes` is it's just the encoded slashes and that other characters will be decoded normally?